modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,70 +4,114 @@ include::_attributes.adoc[]
+The new API version shows the message *This is only visible for you*, showing that the API version visibility is *Admin Only*. Consumer developers viewing the page do not see this API version.
 
-== API updates in Exchange and API Designer
+== Make API Versions Visible
 
-On an API's page in Exchange, the API major version is shown in the dropdown menu next to the API name, and the API asset version is shown in the *Asset versions* list on the right.
+In the ACM administrator interface, select *Manage APIs* at the top.
 
-image::exchange-api-versions.png[Exchange API major version and asset versions]
+Click the *Edit* button to the right of the API version name.
 
-Each API major version can have one or more asset versions.
+Set *Visibility* to *Public*.
 
-ACM imports API major versions and lists them independently of each other, so they can be published and unpublished independently in ACM. However, within each API major version, ACM always uses the latest API asset version, and no other API asset versions.
+Click the *Apply* button at the bottom right.
 
-If a developer publishes a new API asset version by updating the markdown pages in Exchange, or updating the API specification in API Designer, then ACM imports those changes automatically and immediately.
+Consumer developers now see this API version in the API carousel on the community site.
 
-If a developer publishes a new API major version, it will only be published on ACM when the ACM community administrator publishes it manually.
+== Minor API Updates
 
-// Missing so far: Grouping APIs into API Products
+When an API specification or its documentation need corrections such as changing the text, these non-breaking changes are given a minor version change, such as changing version `3.0.0` to version `3.0.1`. ACM automatically and immediately imports these updates to all communities where that API is published, without requiring any action from the ACM administrator.
+
+For example, suppose both the text in an API's documentation and the text in the API Specification description have spelling errors.
+
+To correct errors in the API specification, open Design Center, open the API project, and fix the spelling in the description.
+
+Click the *Publish to Exchange* button at the top right.
+
+In the dialog box *Publish API specification to Exchange*, leave the existing *Name*, *Main file*, and *API Version*, and increment the *Asset version* for a non-breaking change. For example, if the *Asset version* is `3.0.0`, change it to `3.0.1`.
+
+Click the *Publish* button at the bottom right.
+
+To correct errors in the API documentation, open Exchange, open the API, click the *Edit* button at the top right, and fix the errors.
+
+Click the *Save as draft* button at the bottom right.
+
+Click the *Publish* button at the top right.
+
+Consumer developers now see the corrected information on the community site.
+
+== New API Versions
```

Comment:
I like that you explain how versioning works with minor changes. Since theoretically, the version is new for non-breaking changes, maybe name this section something like "Major API Updates." It keeps it consistent conceptually.

Reply:
I agree, "Major API Updates" is better. Fixed.